client/views/actions/channel_list.tpl

```diff
@@ -0,0 +1,18 @@
+<table style="width:100%">
```

Comment:
We should, but this list was very quickly hacked together as a proof of concept, it will need to be improved upon either way.